### DISCUSSION TITLE
Removing enchants

### DISCUSSION THREAD

**Discussion by Ggpepe** [Geolykt/EnchantmentsPlus#37] [Q&A] [answered]
So me and someone run a Minecraft server and we had some problems with op enchants like:
Reaper
quick shot
burst and more 
so is there a way to remove certain enchants

**Comment by Geolykt** [comment, marked as answer]
It's possible by setting the probability of the enchantment to `-1.0`, the `.0` part being very important as otherwise it will lead to unexpected results and is something I still have not yet fixed

**Reply by Ggpepe** [reply]
i think we tried that with bind because it kept duping stuff with death chests and I still got it on some stuff after I made it like that

**Reply by Geolykt** [reply]
It shouldn't function after that though, `0.0` makes something unobtainable, `-1.0` makes it unusable and unobtainable

**Reply by Ggpepe** [reply]
ok i think ill try with the config file later it might work then also
is there a chance if you adding enchants to crossbows and tridents?

**Reply by Ggpepe** [reply]
Wait i set the ones that were not allowed to `-1.0` yet people were able to get it

**Reply by Ggpepe** [reply]
didnt work

**Reply by Geolykt** [reply]
That sounds like a serious bug, I'll try to reproduce later, however I doubt that it is actually there as such a bug would have been reported in the two months since the last update. It is more likely that the enchantment still is in circulation (it won't be removed from circulation) but unusable (which is the main point). Actively removing invalid enchantments would just create too much lag and as such I have opted out of that

About tridents/crossbow, they may be included one day, but given that it would require significant testing it is not something that has top priority.
